bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java

```diff
@@ -19,23 +19,20 @@
 
 public class Messages extends NLS {
```

Comment:
I will fix. I am confused because sometimes Eclipse shows the "missing" messages but for me it didn't :

!ENTRY net.tourbook.common 1 0 2021-02-26 14:44:23.833
!MESSAGE [AppVersion] 20.12.1.qualifier
Warning: NLS missing message: TourChart_PulseGraph_NotAvailable in: net.tourbook.messages
14:44:24.155 [Splash message] Démarrage de l'application...
Application started in : 2902ms
14:44:25.781 [Splash message] Démarrage de la base de données...
14:44:25.520 [Derby command executed] jdbc:derby:tourbook;create=true
14:44:25.520 [Derby command executed] jdbc:derby:tourbook
14:44:25.520 [Derby command executed] jdbc:derby:tourbook
14:44:25.520 [DB Update] Current database design version: 43
14:44:26.492 [Splash message] Démarrage du service de persistance (Hibernate)

Reply:
As you can see, this text key is present in Messages.java but is not translated anywhere, is it normal ?

Warning: NLS missing message: TourChart_PulseGraph_NotAvailable in: net.tourbook.messages

![image](https://user-images.githubusercontent.com/25235665/109317228-bb755600-7809-11eb-925a-c49140877643.png)

Reply:
Ok, I fixed all those issues. It's available in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/308)

![image](https://user-images.githubusercontent.com/25235665/109317755-43f3f680-780a-11eb-9fbf-95adc6931725.png)

Reply:
> 
> 
> @wolfgang-ch
> As you can see, this text key is present in Messages.java but is not translated anywhere, is it normal ?
> 
> Warning: NLS missing message: TourChart_PulseGraph_NotAvailable in: net.tourbook.messages
> 
> ![image](https://user-images.githubusercontent.com/25235665/109317228-bb755600-7809-11eb-925a-c49140877643.png)


Before I do the german translation for a next version, I run these 2 tools on all MT projects, to externalize + cleanup any wrong text keys. The above missing message occur during development when creating text keys but later deleting it in the code.


![mt-text-cleanup](https://user-images.githubusercontent.com/1283445/109378423-b5977780-78d2-11eb-90bd-b4e3cb3a732e.png)